components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go

```diff
@@ -321,6 +346,166 @@ func (nu *NodeUtilsImpl) executeShellCommand(command, path string) error {
 	}, path)
 }
 
+// Stop services on node
+func (nu *NodeUtilsImpl) stopServicesOnNode(ip, nodeType, deploymentType string, infra *AutomateHAInfraDetails) error {
+
+	if nodeType == OPENSEARCH {
+		err := nu.excludeOpenSearchNode(ip, infra)
+		if err != nil {
+			return err
+		}
+	}
+
+	// If deployment type is AWS, then we don't need to stop services on node
+	if deploymentType == AWS_MODE {
+		return nil
+	}
+
+	var cmd string
+	switch nodeType {
+	case AUTOMATE:
+		cmd = STOP_FE_SERVICES_CMD
+	case CHEF_SERVER:
+		cmd = STOP_FE_SERVICES_CMD
+	case POSTGRESQL:
+		cmd = STOP_BE_SERVICES_CMD
+	case OPENSEARCH:
+		cmd = STOP_BE_SERVICES_CMD
+	default:
+		return errors.New("Invalid node type")
+	}
+
+	sshUtil := nu.cmdUtil.GetSshUtil()
+
+	sshUtil.setSSHConfig(&SSHConfig{
+		sshUser:    infra.Outputs.SSHUser.Value,
+		sshPort:    infra.Outputs.SSHPort.Value,
+		sshKeyFile: infra.Outputs.SSHKeyFile.Value,
+		hostIP:     ip,
+	})
+
+	_, err := sshUtil.connectAndExecuteCommandOnRemote(cmd, false)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Also add some writer logs that stopped automate/cs/pg/os node successfully

Reply:
done